### PR TITLE
Implement SDK matrix

### PR DESCRIPTION
**Pull request by ooooolivia**
1) move sdk scenario workitems in ```scenarios.proj ``` to newly created ```sdk_scenarios.proj```
2) make ```scenarios.yml``` take proj file as a parameter so ```scenarios.proj``` and ```sdk_scenarios.proj``` can share the same yml
3) add sdk scenario tests in ```azure-pipelines.yml```
4) implement the **installed sdk vs. tfm** matrix in ```sdk_scenarios.proj```

### CODE REVIEW

**Comment by DrewScoggins** [comment]
Is there a reason we are not adding private jobs for the SDK runs? As it currently stands we won't actually upload any results.

**Comment by DrewScoggins** [comment]
Other than that comment LGTM

**Comment by ooooolivia** [comment]
> Is there a reason we are not adding private jobs for the SDK runs? As it currently stands we won't actually upload any results.

Do you mean 'scheduled job' ? I was thinking adding it after manual testing. But I can add them now.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -320,6 +336,21 @@ jobs:
...
+
+  # Windows x64 SDK scenario benchmarks
+  - template: /eng/performance/scenarios.yml
+    parameters:
+      osName: windows
+      osVersion: RS5
+      architecture: x64
+      pool: Hosted VS2017
+      kind: sdk_scenarios
+      queue: Windows.10.Amd64.19H1.Tiger.Perf
```
Comment (ooooolivia) [inline azure-pipelines.yml]: @DrewScoggins  Maybe you'd want to double-check if the queue is right.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -442,4 +473,24 @@ jobs:
...
```
Comment (ooooolivia) [inline azure-pipelines.yml]: Same here

File: azure-pipelines.yml (hunk 1 above)
Comment (DrewScoggins) [reply azure-pipelines.yml]: Looks right to me.

**Comment by DrewScoggins** [comment]
```[2020/01/09 21:41:04][INFO] $ powershell.exe -NoProfile -ExecutionPolicy Bypass D:\a\1\s\CorrelationStaging\dotnet\dotnet-install.ps1 -InstallDir D:\a\1\s\CorrelationStaging\dotnet -Architecture x64 -Channel master
[2020/01/09 21:41:06][INFO] dotnet-install: Downloading link: https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.100-alpha.1/dotnet-sdk-5.0.100-alpha.1-win-x64.zip
[2020/01/09 21:41:08][INFO] dotnet-install: Cannot download: https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.100-alpha.1/dotnet-sdk-5.0.100-alpha.1-win-x64.zip
[2020/01/09 21:41:08][INFO] dotnet-install: Downloading legacy link: https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.100-alpha.1/dotnet-dev-win-x64.5.0.100-alpha.1.zip
[2020/01/09 21:41:10][INFO] dotnet-install: Cannot download: https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.100-alpha.1/dotnet-dev-win-x64.5.0.100-alpha.1.zip
````

Looks like the dotnet-install script might not be working for the master branch. Maybe we can trigger reruns for the failing legs, see if this was a transient failure or something else.

**Comment by ooooolivia** [comment]
> ```
> [2020/01/09 21:41:06][INFO] dotnet-install: Downloading link: https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.100-alpha.1/dotnet-sdk-5.0.100-alpha.1-win-x64.zip
> [2020/01/09 21:41:08][INFO] dotnet-install: Cannot download: https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.100-alpha.1/dotnet-sdk-5.0.100-alpha.1-win-x64.zip
> [2020/01/09 21:41:08][INFO] dotnet-install: Downloading legacy link: https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.100-alpha.1/dotnet-dev-win-x64.5.0.100-alpha.1.zip
> [2020/01/09 21:41:10][INFO] dotnet-install: Cannot download: https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.100-alpha.1/dotnet-dev-win-x64.5.0.100-alpha.1.zip
> ```
> 
> Looks like the dotnet-install script might not be working for the master branch. Maybe we can trigger reruns for the failing legs, see if this was a transient failure or something else.

@DrewScoggins The script doesn't work for downloading master from master in local runs as well.. The version number is really weird.

**Pull request merged**